src/Functions/Company/AttachmentGet.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @module Intacct/SDK/Functions/Company
+ */
+
+/**
+ * Copyright 2022 Sage Intacct, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import IaXmlWriter from "../../Xml/IaXmlWriter";
+import AbstractFunction from "../AbstractFunction";
+
+export default class AttachmentGet extends AbstractFunction {
```

Comment:
All other `Attachment*` functions extends `AbstractAttachment`. I decided not to do that because it's somewhat incorrect, as not all properties are defined.